pkg/karmadactl/get/get.go

```diff
@@ -275,10 +296,39 @@ func (g *CommandGetOptions) Validate(cmd *cobra.Command) error {
 	return nil
 }
 
+func (g *CommandGetOptions) handleClusterScopeFlags() error {
+	var err error
+	switch g.OperationScope {
+	case string(options.KaramdaControlPlane):
+		g.targetMemberClusters = []string{}
+	case string(options.Members), string(options.All):
+		if len(g.Clusters) == 0 {
+			g.targetMemberClusters, err = ExtractMemberClusters(g.karmadaClient)
+			return err
+		}
+		g.targetMemberClusters = g.Clusters
+		return nil
+	}
+	return nil
+}
+
+func (g *CommandGetOptions) handleNamespaceScopeFlags(f util.Factory) error {
+	var err error
+	g.Namespace, g.ExplicitNamespace, err = f.ToRawKubeConfigLoader().Namespace()
+	if err != nil {
+		return err
+	}
+	if g.AllNamespaces {
+		g.ExplicitNamespace = false
+	}
+	return nil
+}
+
 // Obj cluster info
 type Obj struct {
-	Cluster string
-	Info    *resource.Info
+	Cluster        string
+	IsControlPlane bool
```

Comment:
What's the `IsControlPlane` mean?

Reply:
Used to distinguish whether this cluster is a Karmada control plane